modules/appeals_api/spec/docs/appeals_status/v1_spec.rb

```diff
@@ -99,7 +99,7 @@ def swagger_doc
           end
 
           it 'returns a 422 response' do |example|
-            pending('FIXME: Raw caseflow error is currently returned; Error should be reformatted according to the shared errorModel schema.')
+            pending 'Raw caseflow error is currently returned; Error should be reformatted according to the shared errorModel schema.'
```

Comment:
This spec was failing and breaking the remaining doc generation before this change was made.